src/modules.js

```diff
@@ -311,6 +311,10 @@ function addMissingLibraryStubs() {
 function exportRuntime() {
   const EXPORTED_RUNTIME_METHODS_SET = new Set(EXPORTED_RUNTIME_METHODS);
 
+  const legacyRuntimeElements = new Map();
+  legacyRuntimeElements.set('print', 'out');
+  legacyRuntimeElements.set('printErr', 'err');
```

Comment:
Does JS not have a literal syntax for this?

Reply:
TIL, https://stackoverflow.com/a/35962243. See commit f6c2f803c2085ab474af5c9664b6078c12a4000c.